### PR TITLE
Torch/SolGov content adjustment.

### PR DESCRIPTION
**Pull request by MistakeNot4892**
Generic-ifies the remaining references to the Torch.

### CODE REVIEW

File: code/content_packages/corporate/away_sites/lar_maria/lar_maria.dm [hunk 1]
```diff
@@ -1,3 +1,4 @@
+#include "../../_corporate.dme"
```
Comment (afterthought2) [inline code/content_packages/corporate/away_sites/lar_maria/lar_maria.dm:1]: I don't recall: does this need the `#ifndef BLAH` gating here to make sure it's not included multiple times?

Comment (MistakeNot4892) [reply code/content_packages/corporate/away_sites/lar_maria/lar_maria.dm:1]: `_corporate.dme` gates it itself.

Comment (afterthought2) [reply code/content_packages/corporate/away_sites/lar_maria/lar_maria.dm:1]: ah.

**Review by afterthought2 — APPROVED** [review]

File: code/modules/mechs/mech_wreckage.dm [hunk 2]
```diff
@@ -43,7 +43,7 @@
 		if(WT.isOn())
 			cutting = TRUE
 		else
-			to_chat(user, SPAN_WARNING("Turn the torch on, first."))
+			to_chat(user, SPAN_WARNING("Turn \the [src] on, first."))
```
Comment (comma) [inline code/modules/mechs/mech_wreckage.dm]: shouldn't this be WT, not src?
currently it's "Turn the mecha wreckage first"

**Comment by comma** [comment]
also `maps/random_ruins/exoplanet_ruins/ec_old_crash/` is 100% SCG specific, might want to move it into a pack

**Pull request merged**